packages/rocket-file-uploads-types/src/constants.ts

```diff
@@ -6,3 +6,7 @@ export function azureStorageName(
   const defaultAccountName = `${applicationName}rf${environmentName}`.replace(/([-_])/gi, '').substring(0, 24)
   return storageAccountNameParameter || defaultAccountName
 }
+
+export const LOCAL_PORT_ENV = 'ROCKET_FILES_LOCAL_PORT'
+
+export const localPort = (): string => process.env[LOCAL_PORT_ENV] || '3000'
```

Comment:
I wonder if we need a function and maybe create a constant instead like this:
```suggestion
export const LOCAL_PORT = process.env[LOCAL_PORT_ENV] || '3000'
```

Reply:
Take a look at this comment where we talk about it: https://github.com/boostercloud/rocket-file-uploads/pull/9#discussion_r982281741